pysteps/nowcasts/anvil.py

```diff
@@ -302,9 +286,43 @@ def worker(vil, i):
         starttime_mainloop = time.time()
 
     r_f = []
+
+    if isinstance(timesteps, int):
+        timesteps = range(timesteps + 1)
+        timestep_type = "int"
+    else:
+        original_timesteps = [0] + list(timesteps)
+        timesteps = nowcast_utils.binned_timesteps(original_timesteps)
+        timestep_type = "list"
+
+    if rainrate is not None:
+        r_f_prev = r_vil_a * vil[-1, :] + r_vil_b
+    else:
+        r_f_prev = vil[-1, :]
+    extrap_kwargs["return_displacement"] = True
+
     dp = None
-    for t in range(n_timesteps):
-        print("Computing nowcast for time step %d... " % (t + 1), end="", flush=True)
+    t_nowcast = 0
+    t_prev = 0.0
+
+    for t in range(len(timesteps)):
+        if timestep_type == "list":
+            subtimesteps = [original_timesteps[t_] for t_ in timesteps[t]]
+        else:
+            subtimesteps = [t]
+
+        if len(subtimesteps) > 1 or t > 0:
+            nowcast_time_step = True
+        else:
+            nowcast_time_step = False
+
+        if nowcast_time_step:
+            print(
+                "Computing nowcast for time step %d... " % (t_nowcast + 1),
```

Comment:
`t_nowcast` looks somewhat superfluous, could you simply print `t` instead? 
```suggestion
                "Computing nowcast for time step %d... " % t,
```

Reply:
Done, though I'm not sure if the output is meaningful when a list of time steps is given. In this case, the above prints an info message for each integer time step, but not the actual ones given by the user.

Reply:
Yep, I see your point.. but I guess this is good enough for now..